src/main/resources/db/migration/all/20221014113823__add_assessments_table.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE assessment_schemas (
+    id UUID NOT NULL,
+    added_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    schema TEXT NOT NULL,
+    PRIMARY KEY (id)
+);
+
+INSERT INTO assessment_schemas (id, added_at, "schema") VALUES
```

Comment:
I'm wondering if we should just have one `schemas` table as we've already got a `data` schema for the Applications, as we'll have a `document` schema too, and a `document` schema for assessments too

Reply:
Good shout, have updated